desktop/use-desktop/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Docker Dashboard
+keywords: Docker Dashboard, manage, containers, gui, dashboard, images, user manual
+title: Overview
+redirect_from:
+- /desktop/dashboard/
```

Comment:
See my comment above; this page would then be renamed to `index.md` (to be the index page for the section)

Also note that there's other redirects related to dashboard that were missing, and got added back in https://github.com/docker/docker.github.io/pull/15498; some of those may have to be moved as well